app/src/main/res/values-de/strings.xml

```diff
@@ -1,5 +1,5 @@
 <resources>
-    <string name="app_name">Schlichter Musik-Player</string>
+    <string name="app_name">Simple Music Player</string>
```

Comment:
Why are you reverting app name translations?

Reply:
It's the application name, and I think it's like a persons name. You would not translate it either. But also just a suggestion.

Reply:
app names should be translated too